lib/bpk-component-text/src/BpkText.js

```diff
@@ -141,6 +145,10 @@ const expandAndroidThemedFont = (weight, baseFontFamily) => {
 const getEmphasizeProperties = weight => {
   const emphasizeProperties = {};
 
+  if (weight === 'inherit') {
```

Comment:
Let's move this to the top of the function and just return an empty object. 

```
if (weight === 'inherit') {
  return {};
}
```

Reply:
Changed